aws/resource_aws_dms_certificate_test.go

```diff
@@ -29,9 +29,10 @@ func TestAccAWSDmsCertificate_basic(t *testing.T) {
 				),
 			},
 			{
-				ResourceName:      resourceName,
-				ImportState:       true,
-				ImportStateVerify: true,
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"tags"},
```

Comment:
with `tags` implemented in the `Read` function, we'll be able to verify the argument can be imported so we can remove it from here 👍 
```suggestion
```